lib/components/private/Modal/ariaHideOthers.ts

```diff
@@ -0,0 +1,108 @@
+// Adapted from https://github.com/theKashey/aria-hidden
+
```

Comment:
Lets capture the changes here so we know what is different between the implementations.

Reply:
And we can remove the dep too